.github/CODEOWNERS

```diff
@@ -663,3 +663,5 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 
 # Add owners for notifications for specific pipelines
 /eng/pipelines/aggregate-reports.yml            @AlexGhiondea @maririos
+/sdk/eventhub/tests.data.yml  @serkantkaraca @sjkwak
```

Comment:
These don't have to be at the bottom of the file so you could put them next to the others above just make sure they are the latest entry that matches that file path in the file, so below the other ones that match that path.

Reply:
I chose to put these at the bottom due to the section "# Add owners for notifications for specific pipelines". I will leave it to @AlexGhiondea if he wants to move this around